Update getting started guide links to helm charts (#206)
DataDog/datadog-operator: docs/getting_started.md

> ## Deploy an agent with the operator

Reviewer: ```suggestion
## Deploy the Agent with the operator
```

> In order to deploy a Datadog agent with the operator in the minimum number of steps, the [`datadog-operator`](https://github.com/DataDog/helm-charts/tree/master/charts/datadog-operator) helm chart can be used.

Reviewer: ```suggestion
To deploy the Datadog Agent with the operator in the minimum number of steps, see the [`datadog-operator`](https://github.com/DataDog/helm-charts/tree/master/charts/datadog-operator) helm chart.
```
